src/hooks/index.ts

```diff
@@ -1,3 +1,8 @@
+export {
+	swap,
+	calculateSwap,
+} from "./amm";
+
```

Comment:
Concept of hooks is react specific - let's figure out where to rename the folder which'll hold this logic.